src/Mod/Draft/draftgeoutils/faces.py

```diff
@@ -138,13 +138,46 @@ def create_face(w1, w2):
             and len(w1.Edges) > 1
             and len(w1.Edges) == len(w2.Edges)):
         faces = []
+        facesList = []
         for (edge1, edge2) in zip(w1.Edges, w2.Edges):
-            face = create_face(edge1, edge2)
-            if face is None:
-                return None
-            faces.append(face)
+            from draftgeoutils.intersections import findIntersection
+            # Find potential intersection due to ArchWall Align in opposite
+            # direction (or Edge's Orientation in opposite direction,
+            # or a combinantion of the above)
+            #              v
+            # w1 ----------+
+            #              |
+            # w2 ----------+---------- w1
+            #              |
+            #              +---------- w2
+            # TODO Maybe those edge pair should not be generated in offsetWire().
+            #      Or should have broken into separate wires then.
+            #if DraftGeomUtils.findIntersection(edge1, edge2, dts=False):		# This code fails at some case https://forum.freecad.org/viewtopic.php?p=769210#p769210
+            nearestPts = findIntersection(edge1, edge2, dts=True)			# Works with dts : https://wiki.freecad.org/TopoShape_API : Returns: float<min dist>,list<nearest pts>,list<nearest subshapes & parameters>
```

Comment:
`draftgeoutils.intersections` is the same function as `DraftGeomUtils.findIntersection`. The 'this code fails at some cases' remark cannot be correct.

Reply:
The difference is dts = False or dts = True